autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -4,18 +4,89 @@
 from django.core.management import base
 from scraper.banner_requests import BannerRequests
 from scraper.models import Course, Instructor, Section, Meeting
+from course import generate_course_id
 
-def parse_section(section):
-    " parse_course() must be done first, and pass the data for each section here "
+def parse_section(course):
+    #gets values to create object
+    section_id = ''
+    section_subject = course['subject']
+    course_number = course['courseNumber']
+    section_number = ''
+    section_term_code = course['term']
+    
+    section_min_credits = course['creditHourLow']
+    section_max_credits = course['creditHourHigh']
```

Comment:
This is different between the course and section models. Course just has one field for credit hours but section has 2. Should this be consistent across both?

Reply:
It probably should, seeing as on our website we'll probably want to display the credit hours for a course overall (maybe in the course search, definitely when viewing schedule overviews). As for how we would implement this, whenever adding a section, we could update the corresponding course's minimum credits.